core/engine/src/builtins/temporal/zoneddatetime/mod.rs

```diff
@@ -1152,6 +1558,16 @@ impl ZonedDateTime {
     }
 
     /// 6.3.40 `Temporal.ZonedDateTime.prototype.equals ( other )`
+    ///
+    /// More information:
+    ///
+    /// - [ECMAScript Temporal proposal][spec]
+    /// - [MDN reference][mdn]
+    /// - [`temporal_rs` documentation][temporal_rs-docs]
+    ///
+    /// [spec]: https://tc39.es/proposal-temporal/#sec-temporal.zoneddatetime.prototype.equals
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Temporal/ZonedDateTime/equals
+    /// [temporal_rs-docs]: https://docs.rs/temporal_rs/latest/temporal_rs/struct.ZonedDateTime.html#impl-Eq-for-ZonedDateTime
```

Comment:
nitpick: Maybe point to `PartialEq` where we implement the equality functionality.
```suggestion
    /// [temporal_rs-docs]: https://docs.rs/temporal_rs/latest/temporal_rs/struct.ZonedDateTime.html#impl-PartialEq-for-ZonedDateTime
```